ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -171,12 +174,20 @@ export const CreateGkeApp = ({
           </LinkButton>
         )}
         <CreateAppText />
-        <CreateGkeAppButton
-          {...{ billingStatus, createAppRequest, onDismiss }}
-          existingApp={app}
-          workspaceNamespace={workspace.namespace}
-          username={profile.username}
-        />
+        {app?.status === AppStatus.RUNNING &&
+        isInteractiveUserApp(app?.appType) ? (
```

Comment:
new: if "Open" makes sense (the app is interactive and running), allow the user to do so